spec-draft.md

```diff
@@ -136,6 +136,106 @@ Example payload for setting "Airplane Mode":
 Data is the upper bits since in the future we may want to support setting
 certain types of Data the device is capable of. For example 3G, 4G, LTE.
 
+Device Orientation
+------------------
+
+The [JSON Wire Protocol](https://code.google.com/p/selenium/wiki/JsonWireProtocol#/session/:sessionId/orientation)
```

Comment:
this is essentially a 'bad link' since the project is on github now. New link:

https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidorientation

(yes, i know that page is messed up and needs helping...)